akka-actor/src/main/scala/akka/japi/function/Function.scala

```diff
@@ -55,6 +61,7 @@ trait Predicate[-T] extends java.io.Serializable {
 
 /**
  * A constructor/factory, takes no parameters but creates a new value of type T every call.
+ * Supports throwing exception in the apply, which the `java.util.function.Creator` counterpart does not.
```

Comment:
good

Reply:
not all of them are named `apply` though

Reply:
perhaps use `Exception` since it's only a problem for checked exceptions

Reply:
Ah, good point.